fipy/terms/sourceTerm.py

```diff
@@ -15,6 +15,15 @@ class SourceTerm(CellTerm):
     .. attention:: This class is abstract. Always create one of its subclasses.
     """
     def __init__(self, coeff=0., var=None):
+        r"""
+        Parameters
+        ----------
+        coeff : float or ~fipy.variables.cellVariable.CellVariable
+            Coefficient of source (default: 0)
+        var : ~fipy.variables.cellVariable.CellVariable
+            Variable :math:`\phi` that
+            :class:`~fipy.terms.sourcTerm.SourceTerm` is implicit in.
+        """
         if self.__class__ is SourceTerm:
             raise AbstractBaseClassError
         CellTerm.__init__(self, coeff=coeff, var=var)
```

Comment:
Shouldn't that be using super?

Reply:
Yeah, probably. Certainly should have its typo fixed.

Reply:
Probably. Lots of places should use `super()`. I'm inclined to wait until I dump 2.7 so that I can use the argument-less form.